src/Symfony/Component/Serializer/Tests/Context/Encoder/CsvEncoderContextBuilderTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Tests\Context\Encoder;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Serializer\Context\Encoder\CsvEncoderContextBuilder;
+use Symfony\Component\Serializer\Encoder\CsvEncoder;
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+
+/**
+ * @author Mathias Arlaud <mathias.arlaud@gmail.com>
+ */
+class CsvEncoderContextBuilderTest extends TestCase
+{
+    private CsvEncoderContextBuilder $contextBuilder;
+
+    protected function setUp(): void
+    {
+        $this->contextBuilder = new CsvEncoderContextBuilder();
+    }
+
+    /**
+     * @dataProvider withersDataProvider
+     *
+     * @param array<string, mixed> $values
+     */
+    public function testWithers(array $values)
```

Comment:
missing return type

Reply:
Actually, in the Symfony test suite, no return types are specified, so I followed that guideline.